gui/designwidget.cc

```diff
@@ -487,8 +487,14 @@ void DesignWidget::onClickedBel(BelId bel, bool keep)
         Q_EMIT selected(getDecals(ElementType::BEL, ctx->getBelName(bel)), keep);
     }
     int index = getIndexByElementType(ElementType::BEL);
-    if (tabWidget->currentIndex() != index)
+    if (!keep)
+    {
+        for(int i=0;i<6;i++)

```

Comment:
What's 6? I'd rather have a constant with an appropriate scope and a name that would help me understand what's happening here.

Reply:
…can it be inferred from *selectionModel*?